Saule/Queries/Lambda.cs

```diff
@@ -50,15 +51,27 @@ internal static List<object> TryConvert(List<string> values, Type type)
             return parsedList;
         }
 
+        internal static object TryConvert(string value, Type type)
```

Comment:
I think we need to call this for each filter, so instead of 

```csharp
var parsedValues = TryConvert(values, valueType);
```

(line 17)

```csharp
var parsedValues = values.Select(v => TryConvert(v, valueType)).ToList();
```

Reply:
I had two `TryConvert` methods. One which took an array and one that only took a single one. I have those reduced to one now and added the `Select` function at the appropriate places.